molecule/archive-plain-rhel/verify.yml

```diff
@@ -67,6 +67,113 @@
+
+        update_existing_connector:
+          - name: test-connector-1
+            config:
+              connector.class: "org.apache.kafka.connect.tools.VerifiableSinkConnector"
+              tasks.max: "5"
+              file: "/etc/kafka/connect-distributed.properties"
+              topics: "test_topic"
+
+        add_bad_request_connector:
+          - name: test-connector-1
+            config:
+              connector.class: "org.apache.kafka.connect.tools.VerifiableSinkConnector"
+              tasks.max: "5"
+              file: "/etc/kafka/connect-distributed.properties"
+              topics: "test_topic"
+          - name: test-connector-2
+            config:
+              connector.class: "org.apache.kafka.connect.mirror.MirrorHeartbeatConnector"
+              tasks.max: "1"
+              file: "/etc/kafka/connect-distributed.properties"
+              topics: "foo"
+
+        add_non_existent_connector:
+          - name: test-connector-1
+            config:
+              connector.class: "org.apache.kafka.connect.tools.VerifiableSinkConnector"
+              tasks.max: "5"
+              file: "/etc/kafka/connect-distributed.properties"
+              topics: "test_topic"
+          - name: test-connector-2
+            config:
+              connector.class: "io.confluent.connect.s3.S3SinkConnector"
+              tasks.max: "1"
+              topics: "foo"
+
+        delete_connector:
+          - name: test-connector-2
+            config:
+              connector.class: "org.apache.kafka.connect.tools.VerifiableSinkConnector"
+              tasks.max: "1"
+              topics: "bar"
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_connector.yml
+      vars:
+        connect_url_input: "{{ kafka_connect_url }}"
+        active_connectors_input: "{{ reset_all_connectors }}"
+        expected_message: ""
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_connector.yml
+      vars:
+        connect_url_input: "{{ kafka_connect_url }}"
+        active_connectors_input: "{{ add_new_connector }}"
+        expected_message: "Connectors added or updated: test-connector-1: new connector added."
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_connector.yml
+      vars:
+        connect_url_input: "{{ kafka_connect_url }}"
+        active_connectors_input: "{{ update_existing_connector }}"
+        expected_message: "Connectors added or updated: test-connector-1: connector configuration updated."
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_connector.yml
+      vars:
+        connect_url_input: "{{ kafka_connect_url }}"
+        active_connectors_input: "{{ add_bad_request_connector }}"
+        expected_message: "Connectors added or updated: test-connector-1: no configuration change, test-connector-2: ERROR error while updating configuration (HTTP Error 400: Bad Request)."
```

Comment:
Here, are we trying to add test-connector2 for the first time or updating configurations of an already added test-connector2? 
Might be wrong but I believe it's the former, but the error message suggests it's the latter.

Reply:
checked again, fixed, thanks for pointing!